src/components/Header/styled.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+const HeaderPanel = styled.div`
+  background-color: #e93d44;
```

Comment:
I recommend handling the colors with a theme provider, so changing colors with the light and dark theme becomes easier:

https://styled-components.com/docs/advanced#function-themes

Reply:
I have planed this, but I will implement it in the next deliverables